portfolio/src/main/webapp/script.js

```diff
@@ -16,44 +16,69 @@ window.onload = function() {
     printCommentSender();
 }
 
+
+
 function getComments() {
-    console.log("fetching comments from server");
     document.getElementById("comments").innerHTML = "";
 
     var commentCapacity = document.getElementById("comment-number").value;
     
     fetch("/data?comment-capacity=" + commentCapacity).then(response => response.json()).then((commentsJson) => {
 
-        console.log(commentsJson);
         const commentSectionElement= document.getElementById("comments");
 
         for (comment of commentsJson) {
-            commentSectionElement.appendChild(createCommentElement(comment));
+            console.log(comment);
+            json = JSON.parse(comment);
+            commentSectionElement.appendChild(createcommentElementFromJson(json));
         }
     });
 }
 
-function createCommentElement(text) {
-    const pElement = document.createElement("p");
-    pElement.innerText = text;
-    return pElement;
+function createcommentElementFromJson(json) {
+    
+    var nameElement = document.createElement("b");
```

Comment:
Try to use `const` or `let`: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let